acceptance/internal/config.go

```diff
@@ -100,6 +100,11 @@ type TestConfig struct {
 	// List of keys for which to do string replacement value -> [KEY]. If not set, defaults to true.
 	EnvRepl map[string]bool
 
+	// Name of EnvMatrix key that determines variant-specific output file names.
+	// If set and present in EnvMatrix, update mode will run all variants for this key,
+	// and comparison will only consider files containing the current variant value.
+	EnvOutput *string
```

Comment:
Can we use something more descriptive as maybe `FilterOutputOn`?
`EnvOutput = "DATABRICKS_CLI_DEPLOYMENT"` does not read naturally and takes some time to parse it

Reply:
My impression when reading the `EnvOutput = "DATABRICKS_CLI_DEPLOYMENT"` for the first time and looking at the code was that we define that the result will be written to this DATABRICKS_CLI_DEPLOYMENT variable, or something like this. If you consider it's more fitting, let's leave it like this

Reply:
EnvVaryOutput? If not explanatory, at least should left you with questions what this "vary" is about.